src/huggingface_hub/commands/user.py

```diff
@@ -298,6 +298,24 @@ def run(self):
 </center>"""
 
 
+NOTEBOOK_LOGIN_TOKEN_HTML_START = """<center>
+<img src=https://huggingface.co/front/assets/huggingface_logo-noborder.svg alt='Hugging Face'>
+<br>
+<b>The AI community building the future</b>
+<br>
+Copy a token from <a href="https://huggingface.co/settings/token" target="_blank">you Hugging Face account</a> and paste it below.
+<br>
+Immediately click login after copying your token or it might be stored in plain text in this notebook file.
+</center>"""
```

Comment:
```suggestion
<img src=https://huggingface.co/front/assets/huggingface_logo-noborder.svg alt='Hugging Face'>
<br>
Copy a token from <a href="https://huggingface.co/settings/token" target="_blank">your Hugging Face tokens page</a> and paste it below.
<br>
Immediately click login after copying your token or it might be stored in plain text in this notebook file.
</center>"""
```

Reply:
i think we really need the tagline here, + updated the wording slightly

Other than that LGTM!

Reply:
updated (missing "don't")